example/ck_tile/01_fmha/fmha_fwd.hpp

```diff
@@ -58,6 +58,18 @@ struct FmhaFwdTypeConfig<ck_tile::fp8_t>
     using ODataType           = ck_tile::fp8_t;
 };
 
+struct FmhaDefaultElementFunctions
+{
+    using QElementFunction        = ck_tile::identity;
```

Comment:
can we just leave necessary types for fp8 quantization here?

Reply:
f7d81364f39fc3a9241846130518380e91ea48d2